introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Kathryn, and I am learning to code. Not much else to say at this point. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.